addons/auth/package.json

```diff
@@ -16,8 +16,10 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
```

Comment:
Changes in this file is to reverse changes from when I [upgraded ember for the auth addon](https://github.com/hashicorp/boundary-ui/pull/2776). These changes are still needed.

Reply:
None blocking comment: interesting, I might need to review the core upgrade about these 😅